Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/EditorMeshComponent.cpp

```diff
@@ -112,6 +131,14 @@ namespace AZ
                     ->Attribute(AZ::Script::Attributes::Scope, AZ::Script::Attributes::ScopeFlags::Automation);
 
                 behaviorContext->Class<EditorMeshComponent>()->RequestBus("RenderMeshComponentRequestBus");
```

Comment:
It's interesting that this is prefixed with "renderer".  Was this just to avoid conflict with the old mesh component bus?

Reply:
That is interesting, I don't know why. The Notification bus doesn't do that, so I would expect that to have been an issue if we were dealing with a conflict. Possibly something that was copied from legacy, since we had the concept of a 'RenderMesh'. But your guess is as good as mine.

Might be worth changing. I'm a noob when it comes to behavior context, but I would assume we could add a MeshComponentRequestBus to behavior context that handles the same events, and keep the RenderMeshComponentRequestBus to not break anything existing, then deprecate it over time.